crates/bevy_winit/src/lib.rs

```diff
@@ -254,6 +254,8 @@ struct WinitAppRunnerState {
     active: ActiveState,
     /// Is `true` if a new [`WindowEvent`] has been received since the last update.
     window_event_received: bool,
+    /// Is `true` if a new [`DeviceEvent`] has been received since the last update.
+    device_event_received: bool,
```

Comment:
In a future PR we could replace this with a [`bitbybit`](<https://crates.io/crates/bitbybit>) or [`bilge`](<https://crates.io/crates/bilge>) bitflags To reduce the size of this struct